src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMSource/config.groovy

```diff
@@ -0,0 +1,49 @@
+package com.atlassian.bitbucket.jenkins.internal.scm.BitbucketSCMSource
+
+def f = namespace(lib.FormTagLib)
+def c = namespace(lib.CredentialsTagLib)
+def s = namespace(jenkins.scm.api.FormTagLib)
+
+f.section() {
+
+    f.entry(title: _("bitbucket.scm.credentials"), field: "credentialsId") {
```

Comment:
Probably not worth doing in this PR, but we should separate out the "SCM" section of each of these groovy files (which are identical) into a separate file and import them. That would mean we don't have to change three or files if we need to update documentation or modify/add fields to the SCM

Reply:
agreed. I _think_ @khughes-atlassian tried something like that but it didn't work properly or something. Maybe it was the windows _build_ that failed?